### PR TITLE
feat: implements homeView.sectionsConnection

### PR DESCRIPTION
**Pull request by nickskalkin**
Implements `homeView.sectionsConnection`

### CODE REVIEW

File: src/schema/v2/homeView/index.ts [hunk 1]
```diff
@@ -117,21 +127,45 @@ const SectionType = new GraphQLUnionType({
...
+}).connectionType
+
+const SectionsConnection: GraphQLFieldConfig<any, ResolverContext> = {
+  type: SectionsConnectionType,
+  args: pageable({}),
+  resolve: async (_parent, args, _context, _info) => {
+    const { page, size, offset } = convertConnectionArgsToGravityArgs(args)
+
+    const totalCount = STUB_SECTIONS.length
+    const data = STUB_SECTIONS.slice(offset, offset + size)
```
Comment (nickskalkin) [inline src/schema/v2/homeView/index.ts:148]: I have a suspicion that it won't work with reverse pagination, I will confirm later

Comment (anandaroop) [reply src/schema/v2/homeView/index.ts:148]: Meaning with the `before` arg? Seems fine for now

**Review by anandaroop — COMMENTED** [review]
Looks good, just a small naming change requested 🙏🏽

File: src/schema/v2/homeView/index.ts [hunk 2]
```diff
@@ -117,21 +127,45 @@ const SectionType = new GraphQLUnionType({
...
-// a list (TODO: connection) of sections
-
 const Sections: GraphQLFieldConfig<void, ResolverContext> = {
   type: GraphQLNonNull(GraphQLList(GraphQLNonNull(SectionType))),
   description: "A list of sections on the home view",
   resolve: stubDataResolver,
 }
 
+const SectionsConnectionType = connectionWithCursorInfo({
+  name: "Sections",
```
[the remark this answers is not in the document]
Comment (nickskalkin) [reply src/schema/v2/homeView/index.ts]: @anandaroop done, could you please check if I understood you correctly?

[the remark this answers is not in the document]
Comment (anandaroop) [reply src/schema/v2/homeView/index.ts]: Yep, that's consistent with MP patterns (apparently we prefer a plural field name but a singular type name, as on L167 currently 👍🏽 )

**Pull request merged**